src/Services/CommandHandlerService.cs

```diff
@@ -64,17 +81,23 @@ private async Task OnCommandExecutedAsync(Optional<CommandInfo> command, IComman
                         .Build()
                         .SendToChannel(context.Channel);
                 }
-                else
+                else if (result is ExecuteResult executeResult)
                 {
-                    if (result is ExecuteResult executeResult)
-                    {
-                        await new EmbedBuilder()
-                            .WithColor(Color.Red)
-                            .WithTitle(executeResult.Exception.GetType().Name.Prettify())
-                            .WithDescription(executeResult.Exception.Message)
-                            .Build()
-                            .SendToChannel(context.Channel);
-                    }
+                    await new EmbedBuilder()
+                        .WithColor(Color.Red)
+                        .WithTitle(executeResult.Exception.GetType().Name.Prettify())
+                        .WithDescription(executeResult.Exception.Message)
+                        .Build()
+                        .SendToChannel(context.Channel);
+                }
+                else 
+                {
+                    await new EmbedBuilder()
+                        .WithColor(Color.Red)
+                        .WithDescription("Unknown error! (I did create a log entry for the developers :))")
```

Comment:
Small improvement suggestion, but we could switch `:)` to something like `:smile:` to avoid the double parenthesis.
Also, might want to reword to "I created a log entry for the developers!", and move the actual logging call above the reply.

Reply:
Done